web/geosearch/datapunt_geosearch/db.py

```diff
@@ -16,16 +16,17 @@ def retry_on_psycopg2_error(func):
     """
     @functools.wraps(func)
     def wrapper_retry(*args, **kwargs):
-        retry = 3
-        while retry > 0:
+        retry = 0
+        while retry < 4:
```

Comment:
Ik denk dat `while True` ook zal werken. Bij retry == 4 wordt de error al direct geraised, of je bent eerder al uit de while gebroken (break).

Reply:
`while True` is pure Evil. (c)
----
Huidige aanpak maakt het makkelijker om te snappen wat gebeurd binnen de loop en wanneer loop eindt. Met `while True` weet je niet meer waar is `retry`.